test/unit/app.js

```diff
@@ -0,0 +1,38 @@
+/*
+ This file is a part of libertysoil.org website
+ Copyright (C) 2015  Loki Education (Social Enterprise)
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*eslint-env node, mocha */
+import ga from 'react-google-analytics';
+import { TestUtils, unexpected, expect, React } from '../../test-helpers/expect-unit';
+
+import App from '../../src/pages/app';
+import { initState } from '../../src/store';
+
+
+const GAInitializer = ga.Initializer;
+
+describe('App page it', function() {
+
+  it('should not render GA when process.env.GOOGLE_ANALYTICS_ID not set', function() {
```

Comment:
test DOESN't make sure, that `process.env.GOOGLE_ANALYTICS_ID` actually is not set